docker-compose.yml

```diff
@@ -10,3 +10,4 @@ services:
       - /app/node_modules
     ports:
       - '8080:8080'
+    tty: true
```

Comment:
Required for VS Code devcontainers to start without starting the server. I remove this when I set up docker so I'm just bringing it back